presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -58,6 +58,7 @@ public void testAbs()
         assertFunction("abs(123.45)", DOUBLE, 123.45);
         assertFunction("abs(-123.45)", DOUBLE, 123.45);
         assertFunction("abs(CAST(NULL AS DOUBLE))", DOUBLE, null);
+        assertFunction("abs(FLOAT '-754.1985')", DOUBLE, (double) 754.1985f);
```

Comment:
This and many tests below have to be changed to ensure FLOAT type is returned once scalar functions are PRed (this will happen after merging coercions and Hive support).